debian/snapd.postrm

```diff
@@ -33,17 +43,17 @@ if [ "$1" = "purge" ]; then
             rm -f "/snap/bin/$snap"
             rm -f "/snap/bin/$snap".*
             # snap mount dir
-            umount -l "/snap/$snap/$rev" 2> /dev/null || true
+            umount -d -l "/snap/$snap/$rev" 2> /dev/null || true
```

Comment:
According to the man-page util-linux will be doing the right thing in newer version so a comment that we only have this for trusty compat in here would be nice. We may still need to add the `-d` in `removeMountUnit()` when we remove the snap. If we need the -d here we probably need it also in the go code.